lib/v2/science/maintainer.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
     '/cover': ['y9c', 'TonyRL'],
     '/current/:journal?': ['y9c', 'TonyRL'],
     '/early/:journal?': ['y9c', 'TonyRL'],
+    '/blogs/:name?': ['TomHodson'],
```

Comment:
```suggestion
    '/blogs/:name?': ['TomHodson'],
    '/cover': ['y9c', 'TonyRL'],
    '/current/:journal?': ['y9c', 'TonyRL'],
    '/early/:journal?': ['y9c', 'TonyRL'],
```
Insert new entries in alphabetical order.